core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/pythonUDF/PythonUDFOpExec.java

```diff
@@ -13,7 +13,6 @@
 import org.apache.arrow.flight.*;
 import org.apache.arrow.memory.RootAllocator;
 import org.apache.arrow.vector.VectorSchemaRoot;
-import org.jetbrains.annotations.NotNull;
```

Comment:
is this change related to this PR?

Reply:
this annotation is not available anymore for some reason, also we don't need this annotation